Add automated reminder for API updates (#497)
Shopify/shopify_python_api: .github/api_update_reminder.md

---
title: A new release of the Shopify API is due soon

> assignees: andyw8 # @platform/dev-tools-education

Author: Will be changed to team once finalized.

labels: automated
---

This is an automated reminder for the maintainers that a new Stable release of the Shopify API is due soon, so the library needs to be updated.

A new library release should be prepared to:
* add the upcoming Stable release
* remove the oldest Stable release which will no longer be supported.

The PR should be created as a **draft** but not yet be merged.

The release schedule can be found at https://shopify.dev/concepts/about-apis/versioning

Review the changelog and consider if anything in the library needs to change:

https://shopify.dev/changelog

Test against the upcoming release by using the Release Candidate.

> Another reminder issue will be created on the date of the next release.

Reviewer: Is there any chance we can add a comment to the same issue?

Author: In theory, yes, but I think it might be a bit tricky to script...

Reviewer: Nothing wrong with using a second issue then!

> When that happens, test again using the Stable API version, and aim to release an update of the library within one week.

Reviewer: ```suggestion
When that happens, test again using the now Stable API version, and aim to release an update of the library within one week.
```

Reviewer 2: +1

---
title: A new release of the Shopify API is due soon
assignees: andyw8 # @platform/dev-tools-education
labels: automated
---

This is an automated reminder for the maintainers that a new Stable release of the Shopify API is due soon, so the library needs to be updated.

A new library release should be prepared to:
* add the upcoming Stable release
* remove the oldest Stable release which will no longer be supported.

The PR should be created as a **draft** but not yet be merged.

The release schedule can be found at https://shopify.dev/concepts/about-apis/versioning

Review the changelog and consider if anything in the library needs to change:

https://shopify.dev/changelog

Test against the upcoming release by using the Release Candidate.

> Another reminder on the created on the date of the next release.

Reviewer: ```suggestion
Another reminder will be sent when the Release Candidate is released.
```

> Another reminder on the created on the date of the next release.

Reviewer 2: Slight typo here
